Backpacks.cs

```diff
@@ -666,6 +708,29 @@ public ushort BackpackSize
 
             [JsonProperty("Blacklisted Items (Item Shortnames)")]
             public HashSet<string> BlacklistedItems;
+
+            [JsonProperty(PropertyName = "GUI button")]
+            public GUIButton GUI = new GUIButton();
+
+            public class GUIButton
+            {
+                [JsonProperty(PropertyName = "Enabled?")]
+                public bool Enabled = true;
+
+                [JsonProperty(PropertyName = "Image")]
+                public string Image = "https://i.imgur.com/CyF0QNV.png";
+
+                [JsonProperty(PropertyName = "Background color (RGBA format)")]
+                public string Color = "1 1 1 0.15";
```

Comment:
Would be nice to get a color that's closer to the vanilla UI. I used `1 0.96 0.88 0.15` in another plugin which I remember was fairly close.